vital/algo/algorithm.cxx

```diff
@@ -217,13 +217,13 @@ ::check_nested_algo_configuration( std::string const& type_name,
   if ( ! has_algorithm_impl_name( type_name, instance_name ) )
   {
     std::stringstream msg;
-    msg << "Configuration Failure: invalid option\n"
-        << "   " << type_key << " = " << instance_name << "\n"
-        << "   valid options are";
+    msg << "Implementation '" << instance_name << "' for algorithm type "
+        << type_key << " could not be found.\nMake sure plugin path is set correctly.";
```

Comment:
Maybe more specifically we should say "Make sure KWIVER_PLUGIN_PATH is set correctly"